apps/server/src/modules/common-cartridge/common-cartridge-client/card-client/dto/submission-container-element-content.dto.ts

```diff
@@ -0,0 +1,7 @@
+export class SubmissionContainerElementContentDto {
+	dueDate: string;
```

Comment:
this could be null

Reply:
it's not nullable in the generated model. Should I still change it?

Reply:
yes please, take a look at the original object in ../board/controller/dto/element/submissioncontainerelementcontent.dto.ts

Reply:
Done